asia/README.md

```diff
@@ -1 +1,2 @@
 # ASIA
+[Malaysia](malaysia/readme.md)
```

Comment:
Please, keep the format as the rest of the `README`s, i.e., all of them in capital letters, as the one for Assia and the top level one.